tests/test_cvpr_conference_v2.py

```diff
@@ -740,4 +767,163 @@ def test_secondary_ac_assignment(self, openreview_client, helpers, client):
+                'make_meta_reviews_public': 'No, meta reviews should NOT be revealed publicly when they are posted',
+                'meta_review_start_date': start_date.strftime('%Y/%m/%d'),
+                'meta_review_deadline': due_date.strftime('%Y/%m/%d'),
+                'release_meta_reviews_to_authors': 'No, meta reviews should NOT be revealed when they are posted to the paper\'s authors',
+                'release_meta_reviews_to_reviewers': 'Meta review should not be revealed to any reviewer',
+                'remove_meta_review_form_options': ['recommendation', 'confidence'],
+                'additional_meta_review_form_options': {
+                    "preliminary_recommendation": {
+                        "order": 2,
+                        "description": "Due date: 2024/02/14, 23:59 PT.",
+                        "value": {
+                            "param": {
+                                "type": "string",
+                                "enum": [
+                                    "Clear accept",
+                                    "Needs discussion",
+                                    "Clear reject"
+                                ]
+                            }
+                        }
+                    },
+                    "metareview": {
+                        "order": 1,
+                        "description": "Draft due date: 2024/02/14, 23:59 PT. Final due date: 2024/02/22, 23:59 PT. (Formatting with Markdown and formulas with LaTeX are possible; see https://openreview.net/faq for more information.)",
+                        "value": {
+                            "param": {
+                                "type": "string",
+                                "maxLength": 5000,
+                                "markdown": True,
+                                "input": "textarea"
+                            }
+                        }
+                    }
+                }
+            },
+            forum= request_form.id,
+            invitation= f'openreview.net/Support/-/Request{request_form.number}/Meta_Review_Stage',
+            readers= ['thecvf.com/CVPR/2024/Conference/Program_Chairs', 'openreview.net/Support'],
+            referent= request_form.id,
+            replyto= request_form.id,
+            signatures= ['~Program_CVPRChair1'],
+            writers= [],
+        ))
+
+        helpers.await_queue() 
+
+        # Open meta review revision for final recommendation, allow metareview to be modified
+        due_date = now + datetime.timedelta(days=2)
+        
+        venue.custom_stage = openreview.stages.CustomStage(name='Meta_Review_Revision',
+            reply_to=openreview.stages.CustomStage.ReplyTo.METAREVIEWS,
+            source=openreview.stages.CustomStage.Source.ALL_SUBMISSIONS,
+            reply_type=openreview.stages.CustomStage.ReplyType.REVISION,
+            invitees=[openreview.stages.CustomStage.Participants.AREA_CHAIRS_ASSIGNED],
```

Comment:
should we include the secondary acs as non invitees here?

Reply:
What I did in my PR (https://github.com/openreview/openreview-py/pull/2017), was automatically add secondary ACs as noninvitees if the invitees include ACs but not secondary ACs. 

If you think it's better, I can just support noninvitees as a customStage param?

Reply:
ok, that makes sense. @enrubio can you write an assertion where the secondary AC can not submit a metareview revision for the submission they are secondary ACs?

Reply:
@melisabok @celestemartinez I added a test for this, but I'm not 100% sure it's correct.